freedomofpress/securedrop
# app: add missing journalist templates in apparmor

## Status

Ready for review

## Description of Changes

add missing journalist templates in usr.sbin.apache apparmor

## Testing

* make build-debs
* vagrant up /staging/
* register an admin user
* submit documents
* navigate the journalist interface to see all templates that were added and verify they can be accessed

## Deployment

The updated apparmor file will overwrite the current one.
